doc/doxygen/headers/c++.h

```diff
@@ -54,6 +54,52 @@
  * the idiomatic use of range-based for loops. Examples are
  * GeometryInfo::face_indices(), GeometryInfo::vertex_indices(),
  * FEValuesBase::quadrature_point_indices(), among many others.
+ *
+ * C++11 also introduces the concept of
+ * [constexpr](https://en.cppreference.com/w/cpp/language/constexpr)
```

Comment:
Have you checked that doxygen can render this markup link style appropriately?

Reply:
Yes I checked the html and all of the links inside it.